test/intl402/DateTimeFormat/prototype/formatRangeToParts/argument-to-integer.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2017-2019 André Bargull, Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-partitiondatetimerangepattern
+description: |
+  TimeClip applies ToInteger on its input value.
+info: >
+  Intl.DateTimeFormat.prototype.formatRangeToParts ( startDate , endDate )
+
+  5. Let x be ? ToNumber(startDate).
+  6. Let y be ? ToNumber(endDate).
+
+  TimeClip ( time )
+  ...
+  3. Let clippedTime be ! ToInteger(time).
+  4. If clippedTime is -0, set clippedTime to +0.
+  5. Return clippedTime.
+features: [Intl.DateTimeFormat-formatRange]
+---*/
+
+function* zip(a, b) {
+  assert.sameValue(a.length, b.length);
+  for (let i = 0; i < a.length; ++i) {
+    yield [i, a[i], b[i]];
+  }
+}
+
+function compare(actual, expected) {
+  for (const [i, actualEntry, expectedEntry] of zip(actual, expected)) {
+    assert.sameValue(actualEntry.type, expectedEntry.type, `type for entry ${i}`);
+    assert.sameValue(actualEntry.value, expectedEntry.value, `value for entry ${i}`);
+    assert.sameValue(actualEntry.source, expectedEntry.source, `source for entry ${i}`);
+  }
+}
+
+// Switch to a time format instead of using DateTimeFormat's default date-only format.
+const dtf = new Intl.DateTimeFormat(undefined, {
+    hour: "numeric", minute: "numeric", second: "numeric"
+});
+const date = Date.now();
+const expected = dtf.formatRangeToParts(0, date);
+
+compare(dtf.formatRangeToParts(-0.9, date), expected, "formatToParts(-0.9)");
```

Comment:
the compare function has only 2 args but in here you pass in 3 args
Also the third arg should be "formatRangeToParts" not "formatToParts"